mhs/spineroutelookup/request/tests/test_routing_handler.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+import tornado.testing
+import tornado.web
+from utilities import test_utilities
+
+from request import routing_handler
+from request.tests import test_request_handler
+
+END_POINT_DETAILS = {"end_point": "http://www.example.com"}
+
+
+class TestRoutingRequestHandler(test_request_handler.TestRequestHandler, tornado.testing.AsyncHTTPTestCase):
```

Comment:
Just a general comment but I guess normally we were attempting to avoid multiple inheritance and in theory we should hold test code to the same standard as prod but there's no real risk of any multiple inheritance side effects so I'm not too bothered about this being altered really

Reply:
It's a fair point, and there's not a huge benefit to doing this, so I'll remove the inheritance.